sos/plugins/origin.py

```diff
@@ -146,6 +149,13 @@ def setup(self):
                 self.add_cmd_output("%s logs -n default %s"
                                     % (self.oc_cmd, pod))
 
+            # get a node description from the cluster ndoes
```

Comment:
I believe that this information belongs to the kubernetes plugin.
That plugin already collects node information - not a describe at the moment, but in any case I believe that if we need to amend the information we collect from nodes it should happen in the kubernetes plugin instead.